spec/models/context_spec.rb

```diff
@@ -58,7 +58,7 @@ def method_with_keyword_args positional_arg, non_context_keyword_arg:, user:, sy
     subject { context.can_admin? }
 
     context 'sysadmin' do
-      let(:user) { instance_double 'User', admin_role?: false, sys_admin_role?: true }
+      let(:user) { instance_double User, admin_role?: false, sys_admin_role?: true }
```

Comment:
Do we want both `can_admin?` and `admin_role?`

Reply:
And, nice fix! Do we need an extra test to make sure this works (since we didn't know this was broken before)?